dev/wlp-jakartaee-transform/rules/jakarta-renames.properties

```diff
@@ -89,7 +112,8 @@ javax.xml.bind.helpers=jakarta.xml.bind.helpers
 javax.xml.bind.util=jakarta.xml.bind.util
 javax.xml.bind=jakarta.xml.bind
 javax.xml.ws=jakarta.xml.ws
-
+javax.websocket=jakarta.websocket
+javax.websocket.server=jakarta.websocket.server
```

Comment:
Where are these used?

Reply:
Not sure what' going on here. [These rules are already in integration. ](https://github.com/OpenLiberty/open-liberty/blob/integration/dev/wlp-jakartaee-transform/rules/jakarta-renames.properties#L78-L79)

Since this file is modified numerous times, I think this was lost and picked picked up during a rebase.

Reply:
realized these rules are duplicated, removing